modules/andes-core/broker/src/main/java/org/wso2/andes/store/cassandra/CQLBasedMessageStoreImpl.java

```diff
@@ -882,9 +882,8 @@ public void storeRetainedMessages(List<AndesMessage> retainList) throws AndesExc
     @Override
     public List<String> getAllRetainedTopics() throws AndesException {
         // TODO: implement this method
-        throw new org.apache.commons.lang.NotImplementedException("CQL base message store methods for" +
-                                                                  " retain feature will be implemented " +
-                                                                  "in next iteration");
+        List<String> retainTopicList = Collections.emptyList();
+        return retainTopicList;
```

Comment:
We can directly return a Collections.emptyList() here. I think IntelliJ normally gives a warning here.